.travis.yml

```diff
@@ -6,36 +6,91 @@
 # I would prefer to test against what is in neurodebian
 language: python
 python:
-  - "2.7"
   - "3.5"
   - "3.5-dev"  # 3.5 development branch
   - "3.6"
   - "3.6-dev"  # 3.6 development branch
   - "3.7-dev"  # 3.7 development branch
 env:
-  - VNUMPY=1.8.2
-  - VNUMPY=1.12.1
-  - VNUMPY=1.16.1
+  - VNUMPY=1.8.2  VSCI=0.21.2
+  - VNUMPY=1.8.2  VSCI=0.20.0
+  - VNUMPY=1.8.2  VSCI=0.18
+  - VNUMPY=1.12.1  VSCI=0.21.2
+  - VNUMPY=1.12.1  VSCI=0.20.0
+  - VNUMPY=1.12.1  VSCI=0.18
+  - VNUMPY=1.16.1  VSCI=0.21.2
+  - VNUMPY=1.16.1  VSCI=0.20.0
+  - VNUMPY=1.16.1  VSCI=0.18
+  
 cache:
   - apt
 matrix:
-# old numpy + new python has problems setting up
+  include:
+  - python: "2.7"
+    env: VNUMPY=1.16.1  VSCI=0.12
```

Comment:
Can we add one or two tests with an old numpy as well (e.g. np 1.8 with VSCI .12 and 0.19 if it passes)? That's the last bit from my perspective. Then I think we can merge. Thanks for your work!